common/src/main/webapp/sass/explore-primereact.scss

```diff
@@ -12,21 +12,24 @@
 @import '/lucuma-css/lucuma-ui-prime';
 /* stylelint-enable no-invalid-position-at-import-rule */
 
-// specific to Explore - may be able to remove this when we're fully primereact - may have to apply to tiles or something
-.constraints-tile .explore-tile-body,
-.explore-configuration-grid {
-  background-color: var(--surface-card);
-}
+.dark-theme,
+.light-theme {
```

Comment:
So these had no effect?

Reply:
Unfortunately, not. I still left those on purpose which seemed correct.